app/views/landing/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="landing">
+
+  <!-- Hero -->
+  <div class="landing-hero">
+    <div class="container">
+      <div class="landing-hero-content columns is-justify-content-flex-start">
+        <!-- Text -->
+        <div class="landing-hero-text column is-6-tablet is-7-desktop is-5-widescreen">
+          <div class="landing-hero-flash">
+            <%= render "layouts/flash", isLight: true %>
+          </div>
+
+          <h1 class="landing-hero-title">Tap</h1>
+
+          <div class="landing-hero-description">
+            Yes. We have to drink & eat. But we also have to pay. This handles the drinking part!
```

Comment:
Op GitHub is het:

> Yes. We have to drink. But we also have to pay. This does the drinking part.

Mss dit consistent maken door 1 van beide aan te passen?